apps/client/src/common/utils/dateConfig.ts

```diff
@@ -155,19 +155,21 @@ export const forgivingStringToMillis = (value: string): number => {
   return millis;
 };
 
-export function millisToDelayString(millis: number | null): undefined | string | null {
+export function millisToDelayString(millis: number | null, small = false): undefined | string | null {
```

Comment:
I'm usually not a big fan of having a `boolean` as an optional argument, I feel maybe having "small" | "large" with "large" as default, but its just a nit, maybe its my rails background where the rails linter screams when we do this

Reply:
I believe you and rails are correct. A union flag here would make the code more descriptive